portal/README.md

```diff
@@ -1,30 +1,74 @@
 # Portal
 
+# Prerequisite
+
+1. Need to start `db` and `redis` container
+2. Need to start authgear servers
+
 # Setup locally
 
-We need to setup the environment variable `AUTHGEAR_CLIENT_ID=portal` and `AUTHGEAR_ENDPOINT=http://localhost:3000`.
+## Setup proxy
+
+We need a proxy server for us to call different server with the same domain / port. In `/portal` directory, there is a nginx container for proxy.
```

Comment:
The primary reason of having a proxy is to delegate the authentication, as described [here](https://docs.authgear.com/getting-started/auth-nginx)